<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetDebugInfo</class>
 <widget class="QWidget" name="WidgetDebugInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>555</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>555</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>555</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>555</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>740</width>
     <height>555</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>740</width>
     <height>555</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
	border-image: url(:/images/background/background_debug.png);
}
/* === QScrollBar:vertical === */
QScrollBar:vertical {
    background:transparent;
    width: 16px;
	margin: 0px 0px 0px 0px;
	padding-top:4px;
	padding-bottom:4px;
	
}
QScrollBar::handle:vertical 
{
	background:rgba(255,255,255);
	border-radius:4px;
	width: 8px;
	margin-right:4px; 
	margin-left:4px;
	

}
QScrollBar::handle:vertical:hover
{
    width:8px;
    background:rgba(255,255,255,50%);
    border-radius:4px;
	
    min-height:20;
}


QScrollBar::add-line:vertical {
    background: transparent;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    background: transparent;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}

QScrollBar:up-arrow:vertical, QScrollBar:down-arrow:vertical {
   
    width: 6px;
    height: 6px;
    background:transparent;
}

QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
	background:transparent;
}


</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QTextBrowser" name="textBrowser_debugInfo">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>443</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>443</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextBrowser{
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 18px &quot;.Helvetica Neue DeskInterface&quot;;
border:0px;
}
</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.Helvetica Neue DeskInterface'; font-size:18px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_save">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	border-image: url(:/images/button/button2.png);
	color: rgb(66, 66, 66);
	font: 18px &quot;.Helvetica Neue DeskInterface&quot;;
}
QPushButton:pressed{
border-image: url(:/images/button/button2_press.png);
}
</string>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;</string>
        </property>
        <widget class="QWidget" name="page_normal"/>
        <widget class="QWidget" name="page_success">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 230, 118);
font: 18px &quot;.Helvetica Neue DeskInterface&quot;;</string>
            </property>
            <property name="text">
             <string>保存成功</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_failed">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">color: rgb(244, 67, 54);
font: 18px &quot;.Helvetica Neue DeskInterface&quot;;</string>
            </property>
            <property name="text">
             <string>保存失败</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 18px &quot;.Helvetica Neue DeskInterface&quot;;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_close">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	border-image: url(:/images/button/button2.png);
	color: rgb(66, 66, 66);
	font: 18px &quot;.Helvetica Neue DeskInterface&quot;;
}
QPushButton:pressed{
border-image: url(:/images/button/button2_press.png);
}
</string>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textBrowser_debugInfo</tabstop>
  <tabstop>button_close</tabstop>
  <tabstop>button_save</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
